docs/source/developer/interactives.rst

```diff
@@ -1,126 +1,163 @@
+
+- ``static/interactives/<interactive-slug>/README.md`` - This is a short file that that explains the interactive's purpose and links to any necessary wiki pages.
 
-- `panel.css` - Allows use of the collapsible panels found within the CSFG within an interactive. Use panel HTML structure found within `html-templates.conf`.
-- `third-party/materialize.css` - The default CSS file for [MaterilizeCSS](http://materializecss.com/) (the responsive CSS framework used to create the CSFG). Does not include any custom settings (like our colour scheme):
-  - If the interactive is used `in-page`, do not include link to CSS file within `div` with class `interactive` as every page already has this file loaded (with our custom colour scheme).
-  - The `font` folder with the CSS folder is for use by the Materialize library.
-- `third-party/normalize.css` - Makes browsers render all elements more consistently and in line with modern standards. It precisely targets only the styles that need normalizing. There is no need to use this if using the MaterilizeCSS framework as these defaults are already included.
-- `third-party/foundation.min.css` - The default file for the [Foundation responsive CSS framework](http://foundation.zurb.com/) ([version 5.5.2](http://foundation.zurb.com/sites/docs/v/5.5.3/)). You are welcome to use this for `whole-page` interactives if other frameworks don't suit.
-- `third-party/nouislider.min.css` - Stylesheet for the [noUiSlider library](http://refreshless.com/nouislider/), used within many colour interactives.
-- `third-party/codemirror.css` - Stylesheet for [CodeMirror](https://codemirror.net/), a versatile text editor implemented in JavaScript for the browser. Recommended if you wish to display a text editor within the interactive.
-- `third-party/codemirror-mode-regex.css` - An CSS file for CodeMirror for highlighting regular expression syntax within the editor.
+- ``static/interactives/<interactive-slug>/thumbnail.png`` - ``whole-page`` interactives required a thumbnail image showing the interactive in action.
+  Unless specified when included in the chapter ``md`` file, the file needs to be named as ``thumbnail.png``.
+  The image should be at least 900px wide, and of moderate quality (small size is more important than quality).
 
+- ``templates/interactives/<interactive-slug>.html`` - This is the HTML template for the interactive and it's name must match the slug of the interactive.
+ 
+.. note::
 
-JavaScript Files
+  When creating a new interactive, we suggest you copy the ``static/interactives/example-structure/`` directory and rename it to the slug for your interactive.
+  This directory has the basic structure already mapped out to get you started.
+
+  Similarly, within the ``templates/interactives/`` directory, there is a starter template for each of the interactive types (``iframe-template.html``, ``in-page-template.html`` and ``whole-page-template.html``).
```

Comment:
I was very confused what these templates were until I read this comment (at the end of the PR). It would be wise to prefix all these with something like `example-` or `starter-`.